.changeset/stupid-chicken-itch.md

```diff
@@ -2,4 +2,25 @@
 '@builder.io/qwik-city': minor
 ---
 
-feat: add `valibot$` validator and fix types of `zod$` implementation. To use Valibot, you need to pass `experimental: ['valibot']` as an option to the `qwikVite` plugin.
+feat: **(EXPERIMENTAL)** `valibot$` validator and a fix for `zod$` types.
```

Comment:
```suggestion
FEAT: **(EXPERIMENTAL)** `valibot$` validator and a fix for `zod$` types.
```
Let's align it to other notes

Reply:
@gioboa thanks! but it doesn't matter for changesets, it can be either lowercase or uppercase and it will be replace with an emoji either case :)